manageiq-operator/api/v1alpha1/helpers/miq-components/postgresql_conf.go

```diff
@@ -38,8 +38,10 @@ hot_standby = on
 # ERROR REPORTING AND LOGGING
 #------------------------------------------------------------------------------
 
-log_filename = 'postgresql.log'
-log_rotation_age = 0
+log_destination = 'stderr'
```

Comment:
Technically, this is the default... but I added `stderr` to ensure this is explicit.

Reply:
Any reason not to use stdout?

Reply:
That's not an option in the settings: https://www.postgresql.org/docs/current/runtime-config-logging.html#RUNTIME-CONFIG-LOGGING-WHERE

Ultimately it's probably preferable to use jsonlog, but then I'm not sure you can do jsonlog _and_ do to stderr.

Reply:
It's not an option (stderr, csvlog, jsonlog, and syslog) according to https://www.postgresql.org/docs/current/runtime-config-logging.html#RUNTIME-CONFIG-LOGGING-WHERE.  Perhaps it's to simplify things as `log_collector` "on" would need to redirect both stdout and stderr to logs.

```
log_collector

This parameter enables the logging collector, which is a background process that captures log messages sent to stderr and redirects them into log files
```

Reply:
@Fryguy you beat me to it